WooCommerce/Classes/ViewRelated/Top Banner/TopBannerView.swift

```diff
@@ -179,10 +177,34 @@ private extension TopBannerView {
         return informationStackView
     }
 
-    func createActionStackView() -> UIStackView {
-        let stackView = UIStackView(arrangedSubviews: [actionButton, createBorderView()])
-        stackView.axis = .vertical
-        return stackView
+    func configureActionStackView() {
+        // StackView to hold the action buttons
+        let buttonsStackView = UIStackView()
+        buttonsStackView.distribution = .fillEqually
+        buttonsStackView.spacing = 0.5
+
+        // Background to simulate a separator by giving the buttons some spacing
+        let separatorBackground = createButtonsBackgroundView()
+        buttonsStackView.addSubview(separatorBackground)
+        buttonsStackView.pinSubviewToAllEdges(separatorBackground)
```

Comment:
this is a cool way of showing the separators between arranged subviews :)

Reply:
I think it's crazy that StackView don't support this by default 😅